### PR TITLE
Feature/rn 12.0.1 12.1.1

### PR DESCRIPTION
**Pull request by ThomasMinkeRitense**

### CODE REVIEW

File: release-notes/major12/12.0.1/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Backend libraries 12.1.0
+
+## New Features
+
+The following features were added:
+
+* **Fallback scnario for Keycloak migration script (v12.0)**
+
+  In Valtimo 12.0, a script migrates the assignees for all user tasks from user-email to user-id.
```
Comment (marijnritense) [inline release-notes/major12/12.0.1/valtimo-backend-libraries.md:9]:
```suggestion
  In Valtimo 12.0, a script was introduced that migrates the assignees for all user tasks from user-email to user-id.
```

File: release-notes/major12/12.0.1/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  In the scenario where the assigned user has been removed from Keycloak the migration script would fail, causing the application to crash on startup.
+  In Valtimo 12.1.1, a fallback scenario has been implemented, unassigning the task and logging the original assignee before continuing.
```
Comment (marijnritense) [inline release-notes/major12/12.0.1/valtimo-backend-libraries.md:11]:
```suggestion
  In Valtimo 12.1.1, a fallback scenario has been added, unassigning the task and logging the original assignee before continuing.
```

File: release-notes/major12/12.1.1/valtimo-backend-libraries.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Backend libraries 12.1.1
+
+## New Features
+
+The following features were added:
+
+* **Fallback scnario for Keycloak migration script (v12.0)**
+
+  In Valtimo 12.0, a script migrates the assignees for all user tasks from user-email to user-id.
```
Comment (marijnritense) [inline release-notes/major12/12.1.1/valtimo-backend-libraries.md:9]:
```suggestion
  In Valtimo 12.0, a script was introduced that migrates the assignees for all user tasks from user-email to user-id.
```

File: release-notes/major12/12.0.1/valtimo-backend-libraries.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Backend libraries 12.1.0
+
+## New Features
+
+The following features were added:
+
+* **Fallback scnario for Keycloak migration script (v12.0)**
+
+  In Valtimo 12.0, a script migrates the assignees for all user tasks from user-email to user-id.
+  In the scenario where the assigned user has been removed from Keycloak the migration script would fail, causing the application to crash on startup.
```
Comment (marijnritense) [inline release-notes/major12/12.0.1/valtimo-backend-libraries.md:10]:
```suggestion
  In Valtimo 12.0, a script was introduced that migrates the assignees for all user tasks from user-email to user-id.
```

File: release-notes/major12/12.0.1/valtimo-backend-libraries.md (hunk 1 above)
Comment (marijnritense) [inline release-notes/major12/12.0.1/valtimo-backend-libraries.md:9]:
```suggestion
  In Valtimo 12.0, a script was introduced that migrates the assignees for all user tasks from user-email to user-id.
```

File: release-notes/major12/12.0.1/valtimo-backend-libraries.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Backend libraries 12.1.0
+
+## New Features
+
+The following features were added:
+
+* **Fallback scnario for Keycloak migration script (v12.0)**
```
Comment (Hildebrand-Ritense) [inline release-notes/major12/12.0.1/valtimo-backend-libraries.md:7]: typo scnario -> scenario

File: release-notes/major12/12.1.1/valtimo-backend-libraries.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Backend libraries 12.1.1
+
+## New Features
+
+The following features were added:
+
+* **Fallback scnario for Keycloak migration script (v12.0)**
```
Comment (Hildebrand-Ritense) [inline release-notes/major12/12.1.1/valtimo-backend-libraries.md:7]: typo scnario -> scenario

**Review by marijnritense — APPROVED** [review]

**Pull request merged**